src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,1133 @@
+        srcPtrTemp = srcPtrRow;
+        dstPtrTempR = dstPtrRowR;
+        dstPtrTempG = dstPtrRowG;
+        dstPtrTempB = dstPtrRowB;
+
+        int vectorLoopCount = 0;
+        for (; vectorLoopCount < width; vectorLoopCount += 48)
+        {
+            if(typeid(T) == typeid(Rpp8u))
+            {
+                __m256 p[6];
+                rpp_simd_load(rpp_load48_u8pkd3_to_f32pln3_avx, (Rpp8u *)srcPtrTemp, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_u8pln3_avx, (Rpp8u *)dstPtrTempR, (Rpp8u *)dstPtrTempG, (Rpp8u *)dstPtrTempB, p);
+            }
+            else if(typeid(T) == typeid(Rpp8s)){
+                __m256 p[6];
+                rpp_simd_load(rpp_load48_i8pkd3_to_f32pln3_avx, (Rpp8s *)srcPtrTemp, p);
+                rpp_simd_store(rpp_store48_f32pln3_to_i8pln3_avx, (Rpp8s *)dstPtrTempR, (Rpp8s *)dstPtrTempG, (Rpp8s *)dstPtrTempB, p);
+            }
+            srcPtrTemp += 48;
+            dstPtrTempR += 16;
+            dstPtrTempG += 16;
+            dstPtrTempB += 16;
+        }
+        srcPtrRow += srcDescPtr->strides.hStride;
+        dstPtrRowR += dstDescPtr->strides.hStride;
+        dstPtrRowG += dstDescPtr->strides.hStride;
+        dstPtrRowB += dstDescPtr->strides.hStride;
+    }
+}
+
+RppStatus glitch_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                   RpptDescPtr srcDescPtr,
+                                   Rpp8u *dstPtr,
+                                   RpptDescPtr dstDescPtr,
+                                   Rpp32u *xOffsetR,
+                                   Rpp32u *yOffsetR,
+                                   Rpp32u *xOffsetG,
+                                   Rpp32u *yOffsetG,
+                                   Rpp32u *xOffsetB,
+                                   Rpp32u *yOffsetB,
+                                   RpptROIPtr roiTensorPtrSrc,
+                                   RpptRoiType roiType,
+                                   RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault = {0, 0, (Rpp32s)srcDescPtr->w, (Rpp32s)srcDescPtr->h};
+    omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
+    for (int batchCount = 0; batchCount < dstDescPtr->n; batchCount++)
+    {
+        __m128i maskR = _mm_setr_epi8(0, 0x80, 0x80, 3, 0x80, 0x80, 6, 0x80, 0x80, 9, 0x80, 0x80, 12, 0x80, 0x80, 15);
+        __m128i maskGB = _mm_setr_epi8(0x80, 1, 2, 0x80, 4, 5, 0x80, 7, 8, 0x80, 10, 11, 0x80, 13, 14, 0x80);
+        RpptROI roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
+
+        Rpp32u xOffsetRchn = xOffsetR[batchCount];
+        Rpp32u yOffsetRchn = yOffsetR[batchCount];
+        Rpp32u xOffsetGchn = xOffsetG[batchCount];
+        Rpp32u yOffsetGchn = yOffsetG[batchCount];
+        Rpp32u xOffsetBchn = xOffsetB[batchCount];
+        Rpp32u yOffsetBchn = yOffsetB[batchCount];
+
+        Rpp32u elementsInRowMax = srcDescPtr->w;
```

Comment:
You can directly use srcDescPtr->w
Why again need to reassign in another variable elementsInRowMax?